feed/index.html

```diff
@@ -0,0 +1,164 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="feed page of bingme">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=ADLaM+Display&family=Quicksand&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="../dist/css/style.css">
+    <title>bingme | Feeds </title>
+</head>
+<body>
+    <header class="mb-4">
+        <nav class="navbar navbar-expand-lg">
+            <div class="container">
+                <a class="nav-brand logo fs-2 " href="#" >bing<span>me</span></a>
+                
```

Comment:
Do you want to have a link to the feed page here?

Reply:
i added the link